src/workers/parser/getSymbols.js

```diff
@@ -145,7 +148,7 @@ function extractSymbol(path: SimplePath, symbols) {
   if (t.isClassDeclaration(path)) {
     symbols.classes.push({
       name: path.node.id.name,
-      parent: path.node.superClass,
+      parent: { name: path.node.superClass },
```

Comment:
we had a bug in inComponent that needed this

Reply:
Hmm, this seems to break preview however as the snapshots are not generating properly.

Reply:
I'm going to revert this change, if there's a bug we should open an issue for it. 😄 

<img width="1003" alt="screen shot 2018-06-18 at 9 19 45 am" src="https://user-images.githubusercontent.com/580982/41545617-d210b46a-72d8-11e8-9799-56030c8d65a1.png">